reference/orders.v2.oas2.yml

```diff
@@ -4137,7 +4133,12 @@ components:
           example: false
           type: boolean
         is_tax_inclusive_pricing:
-          description: Indicates whether the order total is inclusive of tax.
+          description: |-
+            Indicate whether the order's base prices are inclusive of tax. 
```

Comment:
```suggestion
Indicate whether the order's base prices include tax.
```

Reply:
hey @bc-traciporter , I've updated this thanks!